src/main/java/com/suse/salt/netapi/client/SaltClient.java

```diff
@@ -409,6 +416,27 @@ public ResultInfoSet getJobResult(final String job) throws SaltException {
         return result.getResult().get(0);
     }
 
+    public <T> Map<String, Result<SSHRawResult>> runRawSSHCommand(final Target<T> target,
```

Comment:
we still need the ssh config here right? also i think it would be better to have the command first and then the target.